RF24Network.h

```diff
@@ -902,6 +907,11 @@ class ESBNetwork
      */
     bool _write(RF24NetworkHeader& header, const void* message, uint16_t len, uint16_t writeDirect);
 
+    /*
+     * The main write function where fragmentation and processing of the payload is initiated
+     */
+    bool main_write(RF24NetworkHeader& header, const void* message, uint16_t len, uint16_t writeDirect);
```

Comment:
Making this private function `inline` does help reduce the compile size. Without it the Priority_TX example's flash size is pretty big (like 4x compile size using inline).
```suggestion
    inline bool main_write(RF24NetworkHeader& header, const void* message, uint16_t len, uint16_t writeDirect);
```